src/rules/isort/mod.rs

```diff
@@ -127,13 +129,82 @@ pub fn format_imports(
     constants: &BTreeSet<String>,
     variables: &BTreeSet<String>,
     no_lines_before: &BTreeSet<ImportType>,
+    forced_separate: &[String],
 ) -> String {
     let trailer = &block.trailer;
     let block = annotate_imports(&block.imports, comments, locator, split_on_trailing_comma);
 
     // Normalize imports (i.e., deduplicate, aggregate `from` imports).
     let block = normalize_imports(block, combine_as_imports);
 
+    let mut output = String::new();
+
+    for block in split::split_by_forced_separate(block, forced_separate) {
```

Comment:
How does this work in the event that we have a forced-separate entry that doesn't end up matching any imports? I think that would return a `Block::default()` corresponding to entry. Do we need to skip empty blocks? Or do they get materialized to the empty string?

Reply:
They get materialized to the empty string – added a configuration to the test that verifies this 👍